tests/artifacts_tests/test_backoff.py

```diff
@@ -3,17 +3,12 @@
 
 from optuna.artifacts import Backoff
 
-from .stubs import FailArtifactStore
-from .stubs import InMemoryArtifactStore
+from tests.artifacts_tests.stubs import FailArtifactStore
+from tests.artifacts_tests.stubs import InMemoryArtifactStore
```

Comment:
IMO, the `tests` module is not intended to be imported by other files.
The primary focus of this repository is to provide the `optuna` module, not the `tests` module.
Could we please keep the relative import within tests?

@nabenabe0928 What do you think?